src/Network/Wai/SAML2/Config.hs

```diff
@@ -38,7 +38,8 @@ data SAML2Config = SAML2Config {
     -- | A value indicating whether to disable time validity checks. This
     -- should not be set to 'True' in a production environment, but may
     -- be useful for testing purposes.
-    saml2DisableTimeValidation :: !Bool
+    saml2DisableTimeValidation :: !Bool,
+    saml2RequireEncryptedAssertion :: !Bool
```

Comment:
There is no documentation for this field.

Reply:
added